pkgs/applications/networking/seadrive-daemon/lib/deblibcurl/default.nix

```diff
@@ -0,0 +1,61 @@
+{ autoPatchelfHook
+, callPackage
+, dpkg
+, fetchurl
+, lib
+, stdenv
+# Package dependencies
+, e2fsprogs
+, krb5
+, libssh2
+, libpsl
+, nghttp2
+, openssl-chacha
+, rtmpdump
+}:
+
+let
+  deblibldap = callPackage ../deblibldap {};
+  deblibssl = callPackage ../deblibssl {};
+in stdenv.mkDerivation rec {
+  version = "7.52.1";
+  pname = "deblibcurl";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "http://ftp.us.debian.org/debian/pool/main/c/curl/libcurl3_7.52.1-5+deb9u9_amd64.deb";
```

Comment:
This kind of url is very short lived. It will disappear in a few weeks. Does nixpkgs's libcurl not work here ?

Reply:
The nixpkgs libcurl is not compatible with the seadrive-daemon binary. Same goes for the other bundled Debian libraries.

Reply:
`seadrive-daemon` gives "no version information available" warnings and runtime errors like "libcurl failed to GET ... SSL peer certificate or SSH remote key was not OK." with nixpkgs curl. I have added this explanation in a comment.